### DISCUSSION TITLE
Deno V2

### DISCUSSION THREAD

**Discussion by ctison** [vercel/turborepo#9246] [Ideas]
### Goals

1. Turborepo is not compatible with Deno V2

### Background

Deno v1 was non-compatible with Node.js but Deno v2 should be[^1]:
> Deno's Node.js compatibility layer means you can bring the best of the Node ecosystem with you to Deno.
[^1]: https://docs.deno.com/runtime/fundamentals/node/

### Proposal

- Allow `deno@2` in `package.json['packageManager']` regexp.
- Parse `deno.json`[^2].

[^2]: https://docs.deno.com/runtime/fundamentals/workspaces/

**Comment by UsamaKhan** [comment]
This was previously brought up in #7454 but, as mentioned, should be easier with the release of v2. @tknickman, would this change the roadmap?

**Comment by anthonyshew** [comment]
We are not currently planning on officially supporting alternative runtimes. It may still work, though, and we're happy to accept PRs that help in support!
